ocaml-lsp-server/src/dune.mli

```diff
@@ -0,0 +1,12 @@
+open Import
+
+type state =
+  | Absent
+  | Out_of_date
+  | Running
```

Comment:
1. I'm finding name `Absent` slightly confusing, as to it's not very clear where it is absent from, especially in the presence of another variant `Running`. Maybe `Not_found` or `Not_found_on_system` (long but very clear)? 
2. Separate error cases from success cases by using `result`?

Reply:
1. `Not_found` sounds good to me.
2. It's worth doing if I use any of the functions in `Result`. I also wouldn't call these error states. Just states where the functionality is diminished.